src/script/conversation/ConversationRepository.ts

```diff
@@ -1927,6 +1940,18 @@ export class ConversationRepository {
     return this.pushToReceivingQueue(eventJson, eventSource);
   };
 
+  private async handleWipeMLSConversation(conversationId: string) {
+    try {
+      const conversation = await this.conversationService.loadConversation<ConversationRecord>(conversationId);
```

Comment:
I think this should be looked in the local state (I think you can try with `this.conversationState.findConversation`

Reply:
Unfortunately it doesn't work in case when my client was logged off, left convo from other client and I login back again. I get `undefined`...